package.json

```diff
@@ -9,6 +9,9 @@
     "lint": "next lint --fix"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.5",
```

Comment:
remove `@emotion`

Reply:
why is not removed? both `@emotion/styled` and `@emotion/react`

Reply:
emotion package is for the material ui